is/utils/io.hpp

```diff
@@ -1,10 +1,64 @@
 #pragma once
 
 #include <string>
+#include <fstream>
 #include "google/protobuf/message.h"
 #include "status.hpp"
+#include "boost/optional.hpp"
+
+namespace google {
```

Comment:
Protobuf is already a public dependency, dont worry about forward declaring this